src/components/profile/ReportingStructure.vue

```diff
@@ -134,4 +149,39 @@ export default {
   font-size: 1em;
   color: var(--gray-50);
 }
+.reporting-structure__directs {
+  min-width: 0;
+}
+.reporting-structure__directs--grid {
+  max-height: 3em;
+  overflow: hidden;
+}
+.reporting-structure__directs--grid .person--avatar-only {
+  margin: 0;
+}
+.reporting-structure__directs--grid-expanded {
+  max-height: none;
+  overflow: visible;
+}
+.reporting-structure__directs--expandable {
+  margin-bottom: 2em;
+}
+.reporting-structure__directs--grid .reporting-structure__directs-list {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 3em);
+  justify-content: space-between;
```

Comment:
Is this intended in case of 3 directs this will spread them across the while space, wouldn't it?

Reply:
You're right, it would and we don't want that; will remove this line, it defaults to `start`.